### PR TITLE
update ledvance.js

### PR DESCRIPTION
**Pull request by dhanukapulkit**

### CODE REVIEW

File: src/devices/ledvance.ts [hunk 1]
```diff
@@ -178,6 +178,22 @@ const definitions: Definition[] = [
         description: 'SMART+ CL B40 E14 Tunable white',
         extend: [ledvanceLight({colorTemp: {range: [153, 370]}})],
     },
+    {
+        zigbeeModel: ['PAR16 TW T'],
+        model: '4058075729162',
+        vendor: 'LEDVANCE',
+        description: 'SMART+ PAR16 GU10 Tunable white',
+        extend: ledvance.extend.light_onoff_brightness_colortemp({colorTempRange: [153, 370]}),
```
Comment (Koenkk) [inline src/devices/ledvance.ts:186]: Remove `ota` and `extend and use:

```
        extend: [ledvanceLight({colorTemp: {range: [153, 370]}})],
```

File: src/devices/ledvance.ts [hunk 2]
```diff
@@ -178,6 +178,22 @@ const definitions: Definition[] = [
...
+        ota: ota.ledvance,
+    },
+    {
+        zigbeeModel: ['B40 TW T'],
+        model: '4058075729087',
+        vendor: 'LEDVANCE',
+        description: 'SMART+ CL B40 E14 Tunable white',
+        extend: ledvance.extend.light_onoff_brightness_colortemp({colorTempRange: [153, 370]}),
```
Comment (Koenkk) [inline src/devices/ledvance.ts:194]: Same here

File: src/devices/ledvance.ts [hunk 3]
```diff
@@ -206,6 +222,14 @@ const definitions: Definition[] = [
         description: 'SMART+ Gardenpole 5P Multicolor',
         extend: [ledvanceLight({colorTemp: {range: [153, 370]}, color: true})],
     },
+    {
+        zigbeeModel: ['GARDENPOLE RGBW T'],
+        model: '4058075729346',
+        vendor: 'LEDVANCE',
+        description: 'SMART+ Gardenpole 5P Multicolor',
+        extend: ledvance.extend.light_onoff_brightness_colortemp_color({colorTempRange: [153, 370]}),
```
Comment (Koenkk) [inline src/devices/ledvance.ts:230]: Remove `ota` and `extend and use:

```
        extend: [ledvanceLight({colorTemp: {range: [153, 370]}, color: true})],
```

File: src/devices/ledvance.ts (hunk 1 above)
Comment (dhanukapulkit) [reply src/devices/ledvance.ts:186]: Done

File: src/devices/ledvance.ts (hunk 2 above)
Comment (dhanukapulkit) [reply src/devices/ledvance.ts:194]: Done

**Comment by dhanukapulkit** [comment]
Made the changes!

**Comment by Koenkk** [comment]
I don't see the changes, if you can give me edit access I can also make them.

**Comment by Koenkk** [comment]
There is still branch protection, so I can't make any more changes, to not waste more time on this I will commit changes directly.

**Comment by dhanukapulkit** [comment]
Thanks @Koenkk  
I will remove the branch protection also. So that it is easier for you too!

**Pull request closed without merging**